nonce/nonce.go

```diff
@@ -28,16 +30,41 @@ import (
 
 	noncepb "github.com/letsencrypt/boulder/nonce/proto"
 	"github.com/prometheus/client_golang/prometheus"
+	"google.golang.org/grpc"
 	"google.golang.org/protobuf/types/known/emptypb"
 )
 
 const (
-	defaultMaxUsed = 65536
-	nonceLen       = 32
+	// PrefixLen is the character length of a nonce prefix.
+	//
+	// TODO(#6610): Remove once we've moved to derivable prefixes by default.
```

Comment:
It looks like all three of PrefixLen, DeprecatedPrefixLen, and NonceService.prefixLen are marked as "TODO: Remove this", at which point there would be no record of the prefix len left at all. I think this one is in error, as PrefixLen is now used by the wfe to set the prefix on the context before calling Redeem.